mkdocs_swan/notebook.html

```diff
@@ -1,4 +1,14 @@
- {% extends "full_width.html" %}
+<script type="text/javascript">
```

Comment:
This js code should be included in some parent block instead of just appended to the top of the file. I believe `extrahead` might be the best one, but not sure.